### ISSUE TITLE
Support `satisfies` operator

### ISSUE THREAD

**Issue by Newbie012** [YousefED/typescript-json-schema#522]
It seems like it's impossible to use this library in conjunction with the new `satisfies` operator. I'm getting errors that look like TS is not aware of the syntax:

```
file.ts: (24,3): ',' expected.
file.ts: Variable 'satisfies' implicitly has an 'any' type.
file.ts: ',' expected.
file.ts: Variable 'WorkflowHumanTaskTemplateDefinition' implicitly has an 'any' type.
file.ts: ',' expected
```

**Comment by netanel-mce** [comment]
I have a same issue 
I use the "ignoreErrors" flag to work around the problem

but @domoritz @YousefED please give us a version with typescript 4.9 +

**Comment by netanel-mce** [comment]
@Newbie012 this bug fixed by PR #529
